src/pydvl/influence/frameworks/__init__.py

```diff
@@ -15,14 +15,17 @@
 
     from .torch_differentiable import (
         as_tensor,
+        cat,
         einsum,
-        identity_tensor,
         mvp,
+        solve_batch_cg,
         solve_linear,
+        solve_lissa,
         stack,
     )
 
     TensorType = torch.Tensor
+    DataLoaderType = torch.utils.data.DataLoader
     ModelType = torch.nn.Module
 
 except ImportError:
```

Comment:
Won't the extension of `__all__` be bogus if the import fails? We are declaring the module to export symbols that are not there. Depending on what other parts of the influence module expects, one should probably fail here with a warning (until there are other frameworks). Or does this break things elsewhere?

Reply:
Should be done